packages/garbo/src/tasks/barfTurn.ts

```diff
@@ -367,6 +389,41 @@ const NonBarfTurnTasks: AlternateTask[] = [
         : 0,
     spendsTurn: true,
   },
+  {
+    name: "Machine Elf Dupe",
+    ready: () =>
+      have($familiar`Machine Elf`) &&
+      // Dupe at end of day even if not ascending, encountersUntilDMTChoice does not reset on rollover
+      willDrunkAdventure() === !sober() &&
+      get("encountersUntilDMTChoice") === 0 &&
+      garboValue(getBestDupeItem()) > get("valueOfAdventure"),
+    completed: () => get("lastDMTDuplication") === myAscensions(),
+    do: $location`The Deep Machine Tunnels`,
+    prepare: () => {
+      if (
+        itemAmount(getBestDupeItem()) < 1 &&
+        closetAmount(getBestDupeItem()) > 0
+      ) {
+        takeCloset(getBestDupeItem());
+      }
```

Comment:
```suggestion
      if (itemAmount(getBestDupeItem()) === 0) retrieveItem(getBestDupeItem());
```
I think `retrieveItem` is better here because it'll do closet _and_ hagnks, for the sickos who have `autoSatisfyWithStorage` flicked on